src/Nancy/Responses/DefaultJsonSerializer.cs

```diff
@@ -100,8 +100,7 @@ private static bool IsJsonType(string contentType)
             return contentMimeType.Equals("application/json", StringComparison.OrdinalIgnoreCase) ||
                 contentMimeType.StartsWith("application/json-", StringComparison.OrdinalIgnoreCase) ||

```

Comment:
Also, just a tiny nit; can we move the `||` before the cases? Like the XML ones.